GeneratedMonitoringArtifacts/Default/DefaultAlerts.json

```diff
@@ -1,438 +0,0 @@
-{
```

Comment:
i see this deleted file is currently referenced by https://aka.ms/azureprometheus-communityalerts from documentation...

Reply:
my bad, I deleted this by mistake. I will add them back and will delete after updating the public doc to point to the new file. Thanks for catching!

Reply:
I reset soft the git commit to put the files back and now the branch got locked out and won't let me push. Created a new pr https://github.com/Azure/prometheus-collector/pull/731 from a new branch. Sorry about that.